src/main/resources/application-integration.properties

```diff
@@ -0,0 +1,42 @@
+transcription.api.url=https://fresenia.man.poznan.pl/dev/tp-api
+server.servlet.context-path=
+europeana.api.tp.authorization-token=wh2B55drz8lGyTDnTYO0nT1Dr6FBKGR0
+# Enabling H2 Console
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+spring.datasource.url=jdbc:h2:~/test;
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.servlet.multipart.max-file-size=100MB
+spring.servlet.multipart.max-request-size=100MB
+spring.jpa.hibernate.naming-strategy=org.hibernate.cfg.ImprovedNamingStrategy
+# Keycloak integration
+keycloak.realm=EnrichEuropeana
+keycloak.auth-server-url=https://sso.apps.paas-dev.psnc.pl/auth
+keycloak.ssl-required=external
+keycloak.public-client=true
+keycloak.principal-attribute=preferred_username
+keycloak.use-resource-role-mappings:true
```

Comment:
should this not be `=` instead of `:`?

Reply:
yes, you right, but it seems to be some legacy bug, as this entire config was just copied in #156 from `application-dev.properties`, which also has this bug. I think we ought to create proper ticket in Jira and change this in future Sprint, as this is not a scope of this task